tests/data/make_butler_repo.py

```diff
@@ -0,0 +1,29 @@
+#!/bin/env python
+# Make a gen3 butler and import the files to have a working cfht_minimal repo.
+#
+# This can be used to regenerate the gen3 registry when the butler has changes
+# that are not backwards compatible.
+#
+# I had to temporarily alter the sdss-dr9-fink-v5b.ecsv file to have absolute
+# paths, because I couldn't get ingest_files with `prefix` no matter what I did
+# on this script, even though it works for the tests.
+
+import lsst.daf.butler
+from lsst.daf.butler.script import ingest_files
+import lsst.obs.base
+
+repopath = "cfht_minimal/repo"
```

Comment:
I couldn't figure it out: the paths in the ecsv file are all relative, so they should have been relative to whatever `prefix=` I passed in to `ingest_files`. I believe that I want the repopath to be relative too, so that everything inside the butler is relative to that (though I can't confirm that changing it the way you suggest would cause butler-internal paths to be absolute).